src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -128,6 +137,15 @@ class ContentSidebar extends React.PureComponent<Props, State> {
         });
 
         this.state = { isLoading: true, isOpen: !!isLarge };
+        /* eslint-disable react/prop-types */
```

Comment:
Still necessary to disable eslint? The other components seem to have it enabled.

Reply:
Yes for some reason it complains here. We actually have a check during the precommit which checks for unused eslint-disable's

Reply:
IMO we should disable this rule for elements folder, as we are not using React.PropTypes